sscpac/statick
# Cpplint error on not found

This fixes #25. When the cpplint tool is not found Statick will now return an error status (`sys.exit(1)`).

To test this I locally did:

    sudo apt-get remove ros-kinetic-roslint

This also uninstalled `ros-kinetic-desktop` and `ros-kinetic-desktop-full`, but for a temporary test this did not cause any problems. Right after testing this issue I was able to easily install all three of those apt packages again.

When running this branch of Statick against `node_example` (with Statick configuration at https://github.com/tdenewiler/node_example/blob/master/statick_config/rsc/config.yaml) I saw that the `cpplint` tool reported that it could not find the executable:

```
Running cpplint tool plugin...
  cpplint not found!
cpplint tool plugin failed
.
.
.
Statick exiting with errors.
```

This resulted in Statick reporting that it was exiting with errors. That should also fail jobs running on continuous integration servers such as Jenkins and Travis.
